### PR TITLE
LPS-156857: Remove unnecessary empty line in continuous set method | 4

### PR DESCRIPTION
**Pull request by qz1010**

### CODE REVIEW

**Comment by qz1010** [comment]
ci:test:sf

**Comment by qz1010** [comment]
ci:test:relevant

File: modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/dao/search/CountrySearchContainerFactory.java [hunk 1]
```diff
@@ -75,7 +75,6 @@ else if (navigation.equals("inactive")) {
 				liferayPortletRequest,
 				AddressPortletKeys.COUNTRIES_MANAGEMENT_ADMIN, "order-by-type",
 				"asc"));
-
```
Comment (ling-alan-huang) [inline modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/dao/search/CountrySearchContainerFactory.java:78]: ??

Comment (qz1010) [reply modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/dao/search/CountrySearchContainerFactory.java:78]: setOrderByType setOrderByComparator 这个是这两个方法中间的空行啊，不应该删除吗

Comment (ling-alan-huang) [reply modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/dao/search/CountrySearchContainerFactory.java:78]: 空行是有意义的，
		searchContainer.setOrderByComparator(
			_getOrderByComparator(
				searchContainer.getOrderByCol(),
				searchContainer.getOrderByType()));

**Comment by qz1010** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
see https://github.com/ling-alan-huang/liferay-portal/pull/1952

**Pull request closed without merging**